src/shared/sleep-config.c

```diff
@@ -142,6 +138,9 @@ int parse_sleep_config(SleepConfig **ret) {
                 }
         }
 
+        if (strv_extend_strv(&sc->modes[SLEEP_HYBRID_SLEEP], sc->modes[SLEEP_HIBERNATE], /* filter_duplicates = */ true) < 0)
+                return log_oom();
```

Comment:
Hmm, if `sc->modes[SLEEP_HYBRID_SLEEP]` is now always the same as ` sc->modes[SLEEP_HIBERNATE]`, then maybe just drop it? What is the point of keeping a separate setting?

Reply:
No? It is initialized to a strv with only `suspend` (check `sleep_default_mode_table`).